modules/comparing-kubernetes-manifests-using-the-roxctl-netpol-connectivity-diff-command.adoc

```diff
@@ -0,0 +1,145 @@
+----
++
+<1> Specify the actual path to the directories containing the Kubernetes manifests you want to compare.
++
+[NOTE]
+====
+The command considers all YAML files that you can accept using `kubectl apply -f`, and then these become valid inputs for your `roxctl netpol connectivity diff` command.
+====
++
+The semantic diff report gives you an overview of the connections that were changed, added, or removed in `dir2` compared to the connections allowed in `dir1`. When you review the output, each line represents one allowed connection that was added, removed, or changed in `dir2` compared to `dir1`.
+If applicable, the `workloads-diff-info` provides additional details about the workloads associated with the added or removed connections.
+
+[id="distinguishing-between-syntactic-and-semantic-diff-outputs_{context}"]
+== Distinguishing between syntactic and semantic diff outputs
+
+In the following example, `dir1` is `netpol-analysis-example-minimal/`, and `dir2` is `netpol-diff-example-minimal/`. The difference between the directories is a small change in the network policy `backend-netpol`.
+
+.Example policy from `dir1`:
+
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  creationTimestamp: null
+  name: backend-netpol
+spec:
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          app: frontend
+    ports:
+    - port: 9090
+      protocol: TCP
+  podSelector:
+    matchLabels:
+      app: backendservice
+  policyTypes:
+  - Ingress
+  - Egress
+status: {}
+----
+
+The change in `dir2` is an added `-` before the ports attribute, which produces a diff output.
+
+
+[id="syntactic-diff-output_{context}"]
+=== Syntactic diff output
+
+.Procedure
+
+* Run the following command to compare the contents of the `netpols.yaml` files in the two specified directories:
++
+----
+$ diff netpol-diff-example-minimal/netpols.yaml netpol-analysis-example-minimal/netpols.yaml
+----
++
+.Example output
++
+[source,terminal,subs="+quotes"]
+----
+12c12
+<   - ports:
+---
+>     ports:
+----
+
+[id="semantic-diff-output_{context}"]
+=== Semantic diff output
+
+.Procedure
+
+* Run the following command to analyze the connectivity differences between the Kubernetes manifests and network policies in the two specified directories:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl netpol connectivity diff --dir1=roxctl/netpol/connectivity/diff/testdata/netpol-analysis-example-minimal/ --dir2=roxctl/netpol/connectivity/diff/testdata/netpol-diff-example-minimal
+----
++
+.Example output
++
+[source,terminal,subs="+quotes"]
+----
+Connectivity diff:
+diff-type: changed, source: default/frontend[Deployment], destination: default/backend[Deployment], dir1:  TCP 9090, dir2: TCP 9090,UDP 53
+diff-type: added, source: 0.0.0.0-255.255.255.255, destination: default/backend[Deployment], dir1:  No Connections, dir2: TCP 9090
+----
+
+[id="connectivity-mapping-using-the-roxctl-connectivity-map-command_{context}"]
```

Comment:
Should this be an H2 heading since it's a different topic than just looking at the connectivity differences? The way I'm reading it, this is looking at all connections together rather than the differences between connections in two versions of files?